modin/engines/base/io/text/csv_dispatcher.py

```diff
@@ -107,7 +107,7 @@ def _read(cls, filepath_or_buffer, **kwargs):
         # Max number of partitions available
         num_partitions = NPartitions.get()
         # This is the number of splits for the columns
-        num_splits = min(len(column_names), num_partitions)
+        num_splits = min(len(column_names) or 1, num_partitions)
```

Comment:
Is this adding really required?

Reply:
Yes, we need to set non-zero `num_splits` value to avoid `ZeroDivision` error [here](https://github.com/modin-project/modin/blob/master/modin/data_management/utils.py#L37).